src/view/com/util/List.web.tsx

```diff
@@ -72,23 +73,35 @@ function ListImpl<ItemT>(
     )
   }
 
-  let header: JSX.Element | null = null
+  const isEmpty = !data || data.length === 0
```

Comment:
All the changes in this file are in a separate PR for @gaearon to review https://github.com/bluesky-social/social-app/pull/4403